### ISSUE TITLE
FProtect Linux passes tools.run_module, but errors through Web Submission

### ISSUE THREAD

**Issue by BrevilleBro** [quarkslab/irma-probe#68]
Running 'python -m tools.run_module FProt eicar.com' as the deploy user returns:

> {'database': None,
>  'duration': 0.844196081161499,
>  'error': None,
>  'name': 'F-PROT Antivirus',
>  'platform': 'linux2',
>  'results': 'EICAR_Test_File (exact)',
>  'status': 1,
>  'type': 'antivirus',
>  'version': '1.0.0'}

However, when I check the probe.log I see the following:

> [2016-06-26 12:26:42,981: DEBUG/Worker-1] probe.tasks.probe_scan[16ef2430-22f7-4b64-bcd0-f2be14147c72]: scanid 5cfc6081-643a-45a8-8965-e8$
$6132s: {**u'status': -1,** u'name': u'F-PROT Antivirus', u'database': None, u'results': None, u'platform': u'linux2', u'version':...

Any ideas?

**Comment by ch0k0bn** [comment]
There is a known problem with FProt and Supervisord, celery dameon launched in commandline are working well but through supervisord fprot process is stuck. Do you launch celery daemon with supervisord (default behaviour for automatic install) ?

**Comment by BrevilleBro** [comment]
Yes, I do. I have used the OVA as a deployment environment, so it should be the standard automatic install. 

Is there a feasible workaround?

**Comment by ch0k0bn** [comment]
You could stop probe celery daemon:
`sudo supervisorctl stop probe_app`

and launch celery daemon manually:

`/opt/irma/irma-probe/current/venv/bin/celery worker --app=probe.tasks:app --workdir=/opt/irma/irma-probe`

For durable workaround, delete supervisord probe_app and check this out: http://irma.readthedocs.io/en/latest/probes/daemon.html#daemon-on-gnu-linux

**Comment by BrevilleBro** [comment]
Thank-you, I will give it a go.

**Comment by BrevilleBro** [comment]
I get an import error if I copy and paste that exact line into Terminal.

> 
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named probe.tasks


I assume I will need to customise the configuration?  Or should that line work with the default environment?

**Comment by BrevilleBro** [comment]
Got it working! 

For me, this line worked:
>/opt/irma/irma-probe/current/venv/bin/celery worker --app=probe.tasks --workdir=/opt/irma/irma-probe/current
